packages/flutter_tools/test/general.shard/android/android_workflow_test.dart

```diff
@@ -338,6 +338,30 @@ Review licenses that have not been accepted (y/N)?
     expect(licenseValidator.runLicenseManager(), throwsToolExit());
   });
 
+  testWithoutContext('AndroidValidator handles a null Android sdkManagerPath', () async {
```

Comment:
I don't think this is testing anything now that `validate` doesn't use `sdkManagerPath` anywhere (unless I'm missing it).

Reply:
sorry, didn't push yet...

Reply:
hmm, even after a push I still see this test on github, but not locally

Reply:
nvm, fixed